### PR TITLE
ci(setup): add app center insights and crash reporting

### PR DESCRIPTION
**Pull request by tplooker**
#### Short description of what this resolves:

Adds basic setup required for app center building and adhoc distribution.

#### Changes proposed in this pull request:

- App center insights and crash reporting
- Removed deprecated android abi 
- Reset build version to 0.1.0 versions

**Fixes**: #22

### CODE REVIEW

File: src/Osma.Mobile.App.Android/Osma.Mobile.App.Android.csproj [hunk 1]
```diff
@@ -63,18 +66,22 @@
   </ItemGroup>
   <ItemGroup />
   <ItemGroup>
-    <AndroidNativeLibrary Include="libs\armeabi\libindy.so" />
```
Comment (Liam-Tait) [inline src/Osma.Mobile.App.Android/Osma.Mobile.App.Android.csproj]: Intentional?

File: src/Osma.Mobile.App/App.xaml.cs [hunk 2]
```diff
@@ -62,7 +65,9 @@ private async Task Initialize()
 
         protected override void OnStart()
         {
-            // Handle when your app starts
+            AppCenter.Start("ios=aaf709cd-8861-4978-87a3-51885163f6a2" +
```
Comment (Liam-Tait) [inline src/Osma.Mobile.App/App.xaml.cs]: App Secret, should this be brought in from environment variables?

File: src/osma-mobile-app.sln [hunk 3]
```diff
@@ -68,8 +68,6 @@ Global
 		{E85A5F7F-DA7D-417F-A8AE-3138C03121DE}.Ad-Hoc|Any CPU.Build.0 = Gorilla|Any CPU
 		{E85A5F7F-DA7D-417F-A8AE-3138C03121DE}.Ad-Hoc|Any CPU.Deploy.0 = Gorilla|Any CPU
 		{E85A5F7F-DA7D-417F-A8AE-3138C03121DE}.Ad-Hoc|iPhone.ActiveCfg = Gorilla|Any CPU
-		{E85A5F7F-DA7D-417F-A8AE-3138C03121DE}.Ad-Hoc|iPhone.Build.0 = Gorilla|Any CPU
```
Comment (Liam-Tait) [inline src/osma-mobile-app.sln:71]: Are these changes intentional?

**Review by Liam-Tait — CHANGES_REQUESTED** [review]

File: src/osma-mobile-app.sln (hunk 3 above)
Comment (tplooker) [reply src/osma-mobile-app.sln:71]: Yeap config not required

File: src/Osma.Mobile.App/App.xaml.cs (hunk 2 above)
Comment (tplooker) [reply src/Osma.Mobile.App/App.xaml.cs]: Yeap just added a prebuild script

File: src/Osma.Mobile.App/App.xaml.cs [hunk 4]
```diff
@@ -62,7 +65,9 @@ private async Task Initialize()
 
         protected override void OnStart()
         {
-            // Handle when your app starts
+            AppCenter.Start("ios=" + AppConstant.IosAnalyticsKey + ";" +
```
Comment (Liam-Tait) [inline src/Osma.Mobile.App/App.xaml.cs]: Should we surround this with a check so it doesn't happen in debug?
```
#if !DEBUG
     AppCenter.Start(
          "ios=" + AppConstant.IosAnalyticsKey + ";" +
          "android=" + AppConstant.AndroidAnalyticsKey + ";",
           typeof(Analytics), typeof(Crashes)
     );
#endif
```

Comment (tplooker) [reply src/Osma.Mobile.App/App.xaml.cs]: Yeap good idea, added

**Review by Liam-Tait — APPROVED** [review]
